test/Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests/Common/TestSuiteCleaner.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.Common;
+
+/* Responsible for all cleanup after the test suite runs -
+* Kills the running docker containers
+* Kills all the processes created
+* Cleans up the used Azure Resources
+*/
+public class TestSuiteCleaner
```

Comment:
Please review the exception handling in this and called classes.

Reply:
We will be taking it up as a separate improvement work item.